app/pages/news/tags/_tag.vue

```diff
@@ -11,38 +11,40 @@
       </header>
       <section class="grid gap-8 lg:grid-cols-3 md:grid-cols-2 sm:grid-cols-1">
         <div v-for="tag in tagsPosts" :key="tag.id" class="">
-          <NewsBlock :news="tag"/>
+          <NewsBlock :news="tag" />
         </div>
       </section>
     </div>
   </div>
 </template>
 
 <script>
-import {contentApiURL, tagsURL} from '~/constants'
+import { contentApiURL, tagsURL } from '~/constants'
 
 export default {
-  data() {
+  data () {
     return {
       tag: this.$route.params.tag,
       tagsPosts: null
     }
   },
-  async fetch() {
+  async fetch () {
     const tagId = await this.$axios.get(`${tagsURL}?slug=${this.tag}`)
       .then(res => res.data[0].id)
       .catch((error) => {
         console.log(error)
-        this.$nuxt.error({statusCode: 500, message: 'Tags not found'})
+        this.$sentry.captureException(new Error('Tags not found ', error))
+        this.$nuxt.error({ statusCode: 500, message: 'Tags not found' })
       })
     this.tagsPosts = await this.$axios.get(`${contentApiURL}/posts?tags=${tagId}&per_page=100`)
       .then(res => res.data)
       .catch((error) => {
         console.log(error)
-        this.$nuxt.error({statusCode: 500, message: 'Tags not found'})
+        this.$sentry.captureException(new Error('Tags posts not found ', error))
+        this.$nuxt.error({ statusCode: 500, message: 'Tags not found' })
```

Comment:
Technically saying not found is correct error but maybe to make it simple for users we could just return `No tags` message or nothing at all, null. 

I think this could be also changed to `statusCode: 404` (or the above error on `L37`?)

Reply:
I don't know the best practices for these messages, I'll rely on your opinion